includes/documents/class-wcpdf-invoice.php

```diff
@@ -333,6 +333,37 @@ public function get_pdf_settings_fields( $option_name ) {
 					),
 				)
 			),
+			array(
+				'type'			=> 'setting',
+				'id'			=> 'due_date',
+				'title'			=> __( 'Display due date', 'woocommerce-pdf-invoices-packing-slips' ),
+				'callback'		=> 'select',
+				'section'		=> $this->type,
+				'args'			=> array(
+					'option_name'	=> $option_name,
+					'id'			=> 'due_date',
+					'options'       => apply_filters( 'wpo_wcpdf_due_date_options', array(
+						''       => __( 'No', 'wpo_wcpdf_pro' ),
+						'1'      => __( '1 day', 'wpo_wcpdf_pro' ),
+						'7'      => __( '7 days', 'wpo_wcpdf_pro' ),
+						'30'     => __( '30 days', 'wpo_wcpdf_pro' ),
+						'custom' => __( 'Custom days', 'wpo_wcpdf_pro' ),
+					) ),
+					'description'	=> __( 'Select a due date to display in the invoice.', 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
I think we need to be more descriptive about the days and where they are added to.

Reply:
What do you think about this sentence:
_Displays a due date below the order data (invoice number, order date, etc.)._

Reply:
That would help yes, but I was thinking about also indication that the days are added to the selected date type.

Reply:
Do you think it might be confusing?
Since the options of the due date are in days, I think it implies that the due date will be calculated based on days. And now that we have a base date (recent commits), it might be clearer.

Reply:
Ok, I agree with that.